### ISSUE TITLE
Client reload doesn't work out-the-box in Chomium

### ISSUE THREAD

**Issue by faustinoaq** [amberframework/amber#1221]
### Description

Chromium Browser doesn't update the code changes in `main.js` or `main.scss`

### Steps to Reproduce

1. `amber watch`
2. Open http://0.0.0.0:3000/ in Chromium Browser
2. Change `$primary` in `main.scss`
3. See triggered reload in Chromium Browser

**Expected behavior:** color changes

**Actual behavior:** colors doesn't change

**Reproduces how often:** 100%

### Versions

```
$ shards --version
Shards 0.10.0 (2020-05-03)
$ crystal -v
Crystal 0.35.0 (2020-06-09)

LLVM: 10.0.0
Default target: x86_64-pc-linux-gnu
$ amber --version
Amber CLI (amberframework.org) - v0.35.0
$ npm -v
6.14.5
$ node -v
v14.3.0
$ lsb_release -a
LSB Version:    n/a
Distributor ID: ManjaroLinux
Description:    Manjaro Linux
Release:        20.0.3
Codename:       Lysia
```

### Additional Information

The issue goes away when I check the option `Disable cache (when DevTools is open)`

![image](https://user-images.githubusercontent.com/3067335/86541072-8cebe900-becf-11ea-812a-dfd7b5ef0b87.png)

I opened this issue to confirm Is this the right workaround or should I change some cache settings in my app?

Happy Crystalling! :heart: :sunglasses:

**Comment by eliasjpr** [comment]
@faustinoaq do you think you can take a look at this?

**Comment by midhun-at-vineti** [comment]
Looks like this has to do with how Chromium and Chrome behave on 'Cache-Control' header, which is set by [pipes/static.cr](https://github.com/amberframework/amber/blob/bb1db0b9536249c7c8e5a6ba9804c3742bd02d9b/src/amber/pipes/static.cr#L134) to private, max-age=3600. 

I tried reproducing this in Chrome but Chrome for some reason doesn't cache the main.bundle.css file but Chromium caches it and is reproducible. 

@eliasjpr @faustinoaq Do you think it would be a good idea to make the static response headers configurable? Would be happy to create a PR.

**Comment by eliasjpr** [comment]
@midhun-at-vineti Yes lets make those headers configurable if possible have some default for Chromium

**Comment by midhun-at-vineti** [comment]
@eliasjpr on it. 👍
